stripper/ze_dreamin_v3_1.cfg

```diff
@@ -10,4 +10,37 @@ modify:
 	{
 		"min_use_angle" "0.2"
 	}
-}
\ No newline at end of file
+}
+
+;Stop Trolling on Stage 1 and 2 endings
```

Comment:
Rather than disabling items that can affect the outcome of a round if a zombie gets close enough, simply cause the particle to be killed off instead, like how 93x's strippers do on some maps: https://github.com/MapTextLang/MapTextLang/blob/master/stripper/ze_FFVII_Cosmo_Canyon_v5k_fix.cfg